package.json

```diff
@@ -7,12 +7,12 @@
     "test": "istanbul cover --report cobertura --config .istanbul.yml -i \"lib/**/*.js\" jasmine-node -- spec --verbose --junitreport --captureExceptions",
     "check": "jasmine-node spec --verbose --junitreport --captureExceptions",
     "gendoc": "jsdoc -r lib -P package.json -R README.md -d .jsdoc",
-    "compile": "babel -s -d lib src",
+    "compile": "npm run lint && babel -s -d lib src",
```

Comment:
is there a reason to maintain a separate `compile` target? why not just stick it into `build` ?